daemon/clients/pods.py

```diff
@@ -0,0 +1,13 @@
+from .peas import AsyncPeaClient
+from .mixin import AsyncToSyncMixin
+
+
+class AsyncPodClient(AsyncPeaClient):
+    """Async Client to create/update/delete Peods on remote JinaD"""
+
+    _kind = 'pod'
+    _endpoint = '/pods'
```

Comment:
Would it maybe be a good idea to reuse the definition from the Api here? So its always in sync?

Reply:
We can move this to enums in a later PR.